### PR TITLE
Rework docs around `connectionSecretName`

### PR DESCRIPTION
**Pull request by JamesGuthrie**

### CODE REVIEW

File: helm-chart/README.md [hunk 1]
```diff
@@ -44,6 +44,35 @@ connection:
...
+  name: my-promscale-connection-secret
+stringData:
+  connection:
+    user: postgres
+    password: ""
+    host: db.timescale.svc.cluster.local
+    port: 5432
+    sslMode: require
+    dbName: timescale
+```
```
Comment (cevian) [inline helm-chart/README.md:68]:
```suggestion
```
In the values.yml of the helm chart you would then have the following:
```

**Review by cevian — APPROVED** [review]
LGTM Small textual suggestion. @paulfantom would know more.

File: helm-chart/README.md (hunk 1 above)
Comment (JamesGuthrie) [reply helm-chart/README.md:67]: done

**Review by cevian — APPROVED** [review]

**Comment by JamesGuthrie** [comment]
FYI I squashed and rebased against latest master

**Review by paulfantom — APPROVED** [review]
lgtm

bors r+

**Pull request merged**